src/main/java/ch/njol/skript/effects/EffReplace.java

```diff
@@ -136,14 +154,17 @@ private void replace(Event event, Object[] needles, Expression<?> haystackExpr)
 					}
 		}
 	}
-	
+
 	@Override
 	public String toString(@Nullable Event event, boolean debug) {
-		if (replaceFirst)
+		if (replaceLast)
+			return "replace last " + needles.toString(event, debug) + " in " + haystack.toString(event, debug) + " with " + replacement.toString(event, debug)
+				+ "(case sensitive: " + caseSensitive + ")";
+		else if (replaceFirst)
 			return "replace first " + needles.toString(event, debug) + " in " + haystack.toString(event, debug) + " with " + replacement.toString(event, debug)
-					+ "(case sensitive: " + caseSensitive + ")";
```

Comment:
You could just use a ternary to add `first` and `last` as needed instead of duplicating the whole line.